server/utils/assessments/placementDurationFromApplication.ts

```diff
@@ -5,7 +5,7 @@ import { retrieveOptionalQuestionResponseFromApplicationOrAssessment } from '../
 
 export const placementDurationFromApplication = (application: Application) => {
   return (
-    retrieveOptionalQuestionResponseFromApplicationOrAssessment(application, PlacementDuration, 'duration') ||
+    Number(retrieveOptionalQuestionResponseFromApplicationOrAssessment(application, PlacementDuration, 'duration')) ||
```

Comment:
This should probs be in the previous commit

Reply:
good catch